os/include/string.h

```diff
@@ -258,6 +265,13 @@ FAR char *strstr(FAR const char *, FAR const char *);
  * @since TizenRT v1.0
  */
 FAR char *strcasestr(FAR const char *, FAR const char *);
+/**
+ * @brief get name of signal
+ * @details @b #include <string.h> \n
+ * POSIX API (refer to : http://pubs.opengroup.org/onlinepubs/9699919799/)
+ * @since TizenRT v2.1 PRE
+ */
+FAR char  *strsignal(int signum);
```

Comment:
Not important but only this has two spaces between return type and function pointer.

Reply:
removed extra space